sql/telemetry_derived/clients_histogram_aggregates_new_v1/query.sql

```diff
@@ -85,16 +85,12 @@ aggregated_histograms AS
       latest_version)
 
 SELECT
+  udf_js.sample_id(client_id) AS sample_id,
```

Comment:
This function seems to work well and it was less code change + less need to pass data around unnecessarily.